index.html

```diff
@@ -188,6 +188,7 @@ <h2>Multi-Post Stroies</h2>
         </div>
       </div>
     </section>
+  </div>
```

Comment:
- [ ] Kindly make sure you have the form on your page. As you can there's no one on your page. :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193203392-115fa395-aa59-4f42-b464-7498a50dff78.png)
 - [ ] Figma design :point_down: 
 
![image](https://user-images.githubusercontent.com/97100091/193203652-8cc30cd0-540b-42dc-970a-3cc6066fe871.png)